nni/common/serializer.py

```diff
@@ -115,7 +115,7 @@ def is_wrapped_with_trace(cls_or_func: Any) -> bool:
     )
 
 
-class SerializableObject(Traceable):
+class SerializableObject(Generic[T], Traceable):
```

Comment:
what is the application scope of `T`, only this file? or the whole code base?

Reply:
For this case, this class.